### DISCUSSION TITLE
Getting Error " The target "MarkupCompilePass1" does not exist in the project"

### DISCUSSION THREAD

**Discussion by SheharyarSMAdil** [microsoft/react-native-windows#10233] [Q&A] [answered]
getting this error while run command `npx react-nartive run-windows`
`- Verbose: ON
i Verbose: ON
Looking for vswhere at: C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe
- Found compatible MSBuild at C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin (16.7.30523.141)
ΓêÜ Found compatible MSBuild at C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin (16.7.30523.141)
- Restoring NuGet packages 
Microsoft (R) Build Engine version 16.7.0+b89cb5fde for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

Building the projects in this solution one at a time. To enable parallel build, please add the "-m" switch.
Build started 02/07/2022 12:14:48.
Project "D:\Office Work\eescom_win\eescom\windows\eescom.sln" on node 1 (Restore target(s)).
ValidateSolutionConfiguration:
  Building solution configuration "Debug|x64".
D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\PropertySheets\JSEngine.props(4,3): warning MSB4011: "D:\Office Work\eescom_win\eescom\windows\ExperimentalFeatures.props" cannot be imported again. It was already imported at "D:\Office Work\eescom_win\eescom\windows\eescom\eescom.vcxproj (4,3)". This is most likely a build authoring error. This subsequent import will be ignored. 
_GetAllRestoreProjectPathItems:
  Determining projects to restore...
Done Building Project "D:\Office Work\eescom_win\eescom\windows\eescom.sln" (Restore target(s)).

Build succeeded.

  D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\PropertySheets\JSEngine.props(4,3): warning MSB4011: "D:\Office Work\eescom_win\eescom\windows\ExperimentalFeatures.props" cannot be imported again. It was already imported at "D:\Office Work\eescom_win\eescom\windows\eescom\eescom.vcxproj (4,3)". This is most likely a build authoring error. This subsequent import will be ignored. 

    1 Warning(s)
    0 Error(s)

Time Elapsed 00:00:00.78
ΓêÜ Restoring NuGet packages 
- Auto-linking...

Parsing project...
Found Windows app project, config:
{
  folder: 'D:\\Office Work\\eescom_win\\eescom',
  sourceDir: 'windows',
  solutionFile: 'eescom.sln',
  project: {
    projectFile: 'eescom\\eescom.vcxproj',
    projectName: 'eescom',
    projectLang: 'cpp',
    projectGuid: '{04cb9512-e177-4418-95a5-4f531ec26ff4}'
  },
  experimentalFeatures: {
    UseHermes: 'false',
    UseWinUI3: 'false',
    UseExperimentalNuget: 'false'
  }
}
Parsing dependencies...
Calculating AutolinkedNativeModules.g.cpp...
Reading AutolinkedNativeModules.g.cpp...
No changes to AutolinkedNativeModules.g.cpp.
Calculating AutolinkedNativeModules.g.props...
Reading AutolinkedNativeModules.g.props...
No changes to AutolinkedNativeModules.g.props.
Calculating AutolinkedNativeModules.g.targets...
Reading AutolinkedNativeModules.g.targets...
No changes to AutolinkedNativeModules.g.targets.
Calculating eescom.sln changes...
ΓêÜ Auto-linking...
Success: No auto-linking changes necessary. (17ms)
- Found Solution: D:\Office Work\eescom_win\eescom\windows\eescom.sln
ΓêÜ Found Solution: D:\Office Work\eescom_win\eescom\windows\eescom.sln
- Build configuration: Debug
i Build configuration: Debug
- Build platform: x64
i Build platform: x64
Running MSBuild with args /clp:NoSummary;NoItemAndPropertyList;Verbosity=normal /nologo /p:Configuration=Debug /p:Platform=x64 /p:AppxBundle=Never /bl /flp1:errorsonly;logfile=C:\Users\Sheha\AppData\Local\Temp\msbuild_24492_build.err /flp2:warningsonly;logfile=C:\Users\Sheha\AppData\Local\Temp\msbuild_24492_build.wrn /restore /p:RestorePackagesConfig=true /p:RunAutolinkCheck=false
- Building Solution
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\msbuild.exe /nologo /bl /clp:NoSummary;NoItemAndPropertyList;Verbosity=normal /flp1:errorsonly;logfile=C:\Users\Sheha\AppData\Local\Temp\msbuild_24492_build.err /flp2:warningsonly;logfile=C:\Users\Sheha\AppData\Local\Temp\msbuild_24492_build.wrn /p:Configuration=Debug /p:Platform=x64 /p:AppxBundle=Never /p:RestorePackagesConfig=true /p:RunAutolinkCheck=false /restore D:\Office Work\eescom_win\eescom\windows\eescom.sln
Building the projects in this solution one at a time. To enable parallel build, please add the "-m" switch.
Build started 02/07/2022 12:14:49.
Project "D:\Office Work\eescom_win\eescom\windows\eescom.sln" on node 1 (Restore target(s)).
ValidateSolutionConfiguration:
  Building solution configuration "Debug|x64".
D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\PropertySheets\JSEngine.props(4,3): warning MSB4011: "D:\Office Work\eescom_win\eescom\windows\ExperimentalFeatures.props" cannot be imported again. It was already imported at "D:\Office Work\eescom_win\eescom\windows\eescom\eescom.vcxproj (4,3)". This is most likely a build authoring error. This subsequent import will be ignored. 
_GetAllRestoreProjectPathItems:
  Determining projects to restore...
Restore:
  Committing restore...
  Committing restore...
  Committing restore...
  Committing restore...
  Assets file has not changed. Skipping assets file writing. Path: D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Folly\obj\project.assets.json
  Assets file has not changed. Skipping assets file writing. Path: D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Microsoft.ReactNative\obj\project.assets.json
  Assets file has not changed. Skipping assets file writing. Path: D:\Office Work\eescom_win\eescom\windows\eescom\obj\project.assets.json
  Restored D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Microsoft.ReactNative\Microsoft.ReactNative.vcxproj (in 75 ms).
  Restored D:\Office Work\eescom_win\eescom\windows\eescom\eescom.vcxproj (in 74 ms).
  Assets file has not changed. Skipping assets file writing. Path: D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\ReactCommon\obj\project.assets.json
  Restored D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\ReactCommon\ReactCommon.vcxproj (in 74 ms).
  Restored D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Folly\Folly.vcxproj (in 75 ms).
  
  NuGet Config files used:
      D:\Office Work\eescom_win\eescom\windows\NuGet.Config
      C:\Users\Sheha\AppData\Roaming\NuGet\NuGet.Config
      C:\Program Files (x86)\NuGet\Config\Microsoft.VisualStudio.Offline.config
  
  Feeds used:
      https://api.nuget.org/v3/index.json
      C:\Program Files (x86)\Microsoft SDKs\NuGetPackages\
  4 of 5 projects are up-to-date for restore.
Done Building Project "D:\Office Work\eescom_win\eescom\windows\eescom.sln" (Restore target(s)).
Project "D:\Office Work\eescom_win\eescom\windows\eescom.sln" on node 1 (default targets).
ValidateSolutionConfiguration:
  Building solution configuration "Debug|x64".
Project "D:\Office Work\eescom_win\eescom\windows\eescom.sln" (1:2) is building "D:\Office Work\eescom_win\eescom\windows\eescom\eescom.vcxproj.metaproj" (8) on node 1 (default targets).
Project "D:\Office Work\eescom_win\eescom\windows\eescom\eescom.vcxproj.metaproj" (8) is building "D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Microsoft.ReactNative\Microsoft.ReactNative.vcxproj" (6:7) on node 1 (default targets).
Project "D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Microsoft.ReactNative\Microsoft.ReactNative.vcxproj" (6:7) is building "D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Common\Common.vcxproj" (7:8) on node 1 (default targets).
GetInstalledSDKLocations:
  Searching for SDKs targeting "UAP, 10.0.19041.0".
  Searching for SDKs targeting "Windows, 8.1".
ResolveSDKReferences:
    Reading SDK manifest file "C:\Program Files (x86)\Microsoft SDKs\Windows Kits\10\ExtensionSDKs\Microsoft.VCLibs\14.0\SDKManifest.xml".
    Targeted configuration and architecture "Debug|x64"
    Could not find "FrameworkIdentity" attribute "FrameworkIdentity-Debug-x64" in the SDK manifest. 
    Found "FrameworkIdentity" attribute "Name = Microsoft.VCLibs.140.00.Debug, MinVersion = 14.0.27810.0, Publisher = 'CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US'" in the SDK manifest.
    Found "APPX" location attribute "AppX-Debug-x86=.\AppX\Debug\x86\Microsoft.VCLibs.x86.Debug.14.00.appx" in the SDK manifest.
    Found "APPX" location attribute "AppX-Debug-x64=.\AppX\Debug\x64\Microsoft.VCLibs.x64.Debug.14.00.appx" in the SDK manifest.
    Found "APPX" location attribute "AppX-Debug-ARM=.\AppX\Debug\ARM\Microsoft.VCLibs.ARM.Debug.14.00.appx" in the SDK manifest.
    Found "APPX" location attribute "AppX-Debug-ARM64=.\AppX\Debug\ARM64\Microsoft.VCLibs.ARM64.Debug.14.00.appx" in the SDK manifest.
ExpandSDKReferences:
  Enumerating SDK Reference "Microsoft.VCLibs, Version=14.0" from "C:\Program Files (x86)\Microsoft SDKs\Windows Kits\10\ExtensionSDKs\Microsoft.VCLibs\14.0\".
InitializeBuildStatus:
  Touching "build\x64\Debug\Common\Common.tlog\unsuccessfulbuild".
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppBuild.targets(917,11): error MSB4057: The target "MarkupCompilePass1" does not exist in the project. [D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Common\Common.vcxproj]
Done Building Project "D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Common\Common.vcxproj" (default targets) -- FAILED.
Project "D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Microsoft.ReactNative\Microsoft.ReactNative.vcxproj" (6:7) is building "D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Folly\Folly.vcxproj" (3:9) on node 1 (default targets).
Project "D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Folly\Folly.vcxproj" (3:9) is building "D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Fmt\Fmt.vcxproj" (4:8) on node 1 (default targets).
GetInstalledSDKLocations:
  Searching for SDKs targeting "UAP, 10.0.19041.0".
  Searching for SDKs targeting "Windows, 8.1".
ResolveSDKReferences:
    Reading SDK manifest file "C:\Program Files (x86)\Microsoft SDKs\Windows Kits\10\ExtensionSDKs\Microsoft.VCLibs\14.0\SDKManifest.xml".
    Targeted configuration and architecture "Debug|x64"
    Could not find "FrameworkIdentity" attribute "FrameworkIdentity-Debug-x64" in the SDK manifest. 
    Found "FrameworkIdentity" attribute "Name = Microsoft.VCLibs.140.00.Debug, MinVersion = 14.0.27810.0, Publisher = 'CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US'" in the SDK manifest.
    Found "APPX" location attribute "AppX-Debug-x86=.\AppX\Debug\x86\Microsoft.VCLibs.x86.Debug.14.00.appx" in the SDK manifest.
    Found "APPX" location attribute "AppX-Debug-x64=.\AppX\Debug\x64\Microsoft.VCLibs.x64.Debug.14.00.appx" in the SDK manifest.
    Found "APPX" location attribute "AppX-Debug-ARM=.\AppX\Debug\ARM\Microsoft.VCLibs.ARM.Debug.14.00.appx" in the SDK manifest.
    Found "APPX" location attribute "AppX-Debug-ARM64=.\AppX\Debug\ARM64\Microsoft.VCLibs.ARM64.Debug.14.00.appx" in the SDK manifest.
ExpandSDKReferences:
  Enumerating SDK Reference "Microsoft.VCLibs, Version=14.0" from "C:\Program Files (x86)\Microsoft SDKs\Windows Kits\10\ExtensionSDKs\Microsoft.VCLibs\14.0\".
InitializeBuildStatus:
  Touching "build\x64\Debug\fmt\fmt.tlog\unsuccessfulbuild".
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppBuild.targets(917,11): error MSB4057: The target "MarkupCompilePass1" does not exist in the project. [D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Fmt\Fmt.vcxproj]
Done Building Project "D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Fmt\Fmt.vcxproj" (default targets) -- FAILED.
Done Building Project "D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Folly\Folly.vcxproj" (default targets) -- FAILED.
Done Building Project "D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Microsoft.ReactNative\Microsoft.ReactNative.vcxproj" (default targets) -- FAILED.
Done Building Project "D:\Office Work\eescom_win\eescom\windows\eescom\eescom.vcxproj.metaproj" (default targets) -- FAILED.
Done Building Project "D:\Office Work\eescom_win\eescom\windows\eescom.sln" (default targets) -- FAILED.
Deferred Messages
  
  Detailed Build Summary
  ======================
      
  
  ============================== Build Hierarchy (IDs represent configurations) =====================================================
  Id                  : Exclusive Time   Total Time   Path (Targets)
  -----------------------------------------------------------------------------------------------------------------------------------
  0                   : 0.416s           1.618s       D:\Office Work\eescom_win\eescom\windows\eescom.sln (Restore) 
  | 12                : 0.000s           0.000s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Common\Common.vcxproj (_GenerateRestoreProjectPathWalk) 
  | 11                : 0.000s           0.000s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Microsoft.ReactNative\Microsoft.ReactNative.vcxproj (_GenerateRestoreProjectPathWalk) 
  | 10                : 0.000s           0.000s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\ReactCommon\ReactCommon.vcxproj (_GenerateRestoreProjectPathWalk) 
  | 9                 : 0.000s           0.000s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\fmt\fmt.vcxproj (_GenerateRestoreProjectPathWalk) 
  | 8                 : 0.000s           0.000s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Folly\Folly.vcxproj (_GenerateRestoreProjectPathWalk) 
  | 1                 : 0.218s           0.218s       D:\Office Work\eescom_win\eescom\windows\eescom\eescom.vcxproj (_IsProjectRestoreSupported) 
  | 2                 : 0.074s           0.074s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Folly\Folly.vcxproj (_IsProjectRestoreSupported) 
  | 3                 : 0.051s           0.051s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\fmt\fmt.vcxproj (_IsProjectRestoreSupported) 
  | 4                 : 0.076s           0.076s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\ReactCommon\ReactCommon.vcxproj (_IsProjectRestoreSupported) 
  | 5                 : 0.162s           0.162s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Microsoft.ReactNative\Microsoft.ReactNative.vcxproj (_IsProjectRestoreSupported) 
  | 6                 : 0.057s           0.057s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Common\Common.vcxproj (_IsProjectRestoreSupported) 
  | 7                 : 0.017s           0.152s       D:\Office Work\eescom_win\eescom\windows\eescom\eescom.vcxproj (_GenerateRestoreProjectPathWalk) 
  | | 12              : 0.010s           0.010s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Common\Common.vcxproj (_GenerateRestoreProjectPathWalk) 
  | | 9               : 0.010s           0.010s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\fmt\fmt.vcxproj (_GenerateRestoreProjectPathWalk) 
  | | 8               : 0.015s           0.015s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Folly\Folly.vcxproj (_GenerateRestoreProjectPathWalk) 
  | . 11              : 0.078s           0.100s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Microsoft.ReactNative\Microsoft.ReactNative.vcxproj (_GenerateRestoreProjectPathWalk) 
  | | . 10            : 0.021s           0.021s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\ReactCommon\ReactCommon.vcxproj (_GenerateRestoreProjectPathWalk) 
  | 13                : 0.085s           0.085s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Common\Common.vcxproj (_GenerateRestoreGraphProjectEntry) 
  | 14                : 0.019s           0.019s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\fmt\fmt.vcxproj (_GenerateRestoreGraphProjectEntry) 
  | 15                : 0.019s           0.019s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Folly\Folly.vcxproj (_GenerateRestoreGraphProjectEntry) 
  | 16                : 0.082s           0.082s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Microsoft.ReactNative\Microsoft.ReactNative.vcxproj (_GenerateRestoreGraphProjectEntry) 
  | 17                : 0.022s           0.022s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\ReactCommon\ReactCommon.vcxproj (_GenerateRestoreGraphProjectEntry) 
  | 18                : 0.019s           0.019s       D:\Office Work\eescom_win\eescom\windows\eescom\eescom.vcxproj (_GenerateRestoreGraphProjectEntry) 
  | 19                : 0.018s           0.018s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Common\Common.vcxproj (_GenerateProjectRestoreGraph) 
  | 20                : 0.011s           0.011s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\fmt\fmt.vcxproj (_GenerateProjectRestoreGraph) 
  | 21                : 0.019s           0.019s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Folly\Folly.vcxproj (_GenerateProjectRestoreGraph) 
  | 22                : 0.082s           0.082s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Microsoft.ReactNative\Microsoft.ReactNative.vcxproj (_GenerateProjectRestoreGraph) 
  | 23                : 0.020s           0.020s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\ReactCommon\ReactCommon.vcxproj (_GenerateProjectRestoreGraph) 
  . 24                : 0.016s           0.016s       D:\Office Work\eescom_win\eescom\windows\eescom\eescom.vcxproj (_GenerateProjectRestoreGraph) 
  
  ============================== Node Utilization (IDs represent configurations) ====================================================
  Timestamp:            1        Duration   Cumulative
  -----------------------------------------------------------------------------------------------------------------------------------
  637923410899472373:   0        0.086s     0.086s .
  637923410900333942:   1        0.218s     0.304s ....
  637923410902513947:   2        0.074s     0.378s .
  637923410903249671:   3        0.051s     0.429s .
  637923410903760291:   4        0.076s     0.505s .
  637923410904520684:   5        0.162s     0.667s ...
  637923410906140660:   6        0.057s     0.723s .
  637923410906706577:   0        0.009s     0.732s 
  637923410906794680:   7        0.016s     0.748s 
  637923410906954930:   12       0.010s     0.758s 
  637923410907054826:   9        0.010s     0.768s 
  637923410907154554:   8        0.015s     0.783s 
  637923410907304984:   11       0.077s     0.860s .
  637923410908070135:   10       0.021s     0.881s 
  637923410908280116:   11       0.002s     0.883s 
  637923410908300126:   7        0.001s     0.884s 
  637923410908310123:   0        0.006s     0.890s 
  637923410908370118:   13       0.085s     0.974s .
  637923410909216586:   14       0.019s     0.993s 
  637923410909406574:   15       0.019s     1.012s 
  637923410909596489:   16       0.082s     1.094s .
  637923410910416570:   17       0.022s     1.116s 
  637923410910632660:   18       0.019s     1.135s 
  637923410910825705:   0        0.001s     1.136s 
  637923410910835445:   19       0.018s     1.155s 
  637923410911017863:   20       0.011s     1.166s 
  637923410911127953:   21       0.020s     1.186s 
  637923410911327571:   22       0.082s     1.268s .
  637923410912149437:   23       0.020s     1.288s 
  637923410912350002:   24       0.016s     1.304s 
  637923410912509413:   0        0.314s     1.618s ......
  -----------------------------------------------------------------------------------------------------------------------------------
  Utilization:          100.0    Average Utilization: 100.0
  
  Detailed Build Summary
  ======================
      
  
  ============================== Build Hierarchy (IDs represent configurations) =====================================================
  Id                  : Exclusive Time   Total Time   Path (Targets)
  -----------------------------------------------------------------------------------------------------------------------------------
  25                  : 0.006s           0.741s       D:\Office Work\eescom_win\eescom\windows\eescom.sln () 
  | 26                : 0.007s           0.733s       D:\Office Work\eescom_win\eescom\windows\eescom\eescom.vcxproj.metaproj () 
  | . 28              : 0.282s           0.726s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Microsoft.ReactNative\Microsoft.ReactNative.vcxproj () 
  | | | 29            : 0.189s           0.189s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Common\Common.vcxproj () 
  | | | 30            : 0.090s           0.184s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Folly\Folly.vcxproj () 
  | | | . 32          : 0.094s           0.094s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Fmt\Fmt.vcxproj () 
  | | . 31            : 0.071s           0.071s       D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\ReactCommon\ReactCommon.vcxproj () 
  . 27                : 0.002s           0.002s       D:\Office Work\eescom_win\eescom\windows\..\node_modules\react-native-windows\ReactCommon\ReactCommon.vcxproj.metaproj () 
  
  ============================== Node Utilization (IDs represent configurations) ====================================================
  Timestamp:            1        Duration   Cumulative
  -----------------------------------------------------------------------------------------------------------------------------------
  637923410899472373:   0        0.086s     0.086s .
  637923410900333942:   1        0.218s     0.304s ....
  637923410902513947:   2        0.074s     0.378s .
  637923410903249671:   3        0.051s     0.429s .
  637923410903760291:   4        0.076s     0.505s .
  637923410904520684:   5        0.162s     0.667s ...
  637923410906140660:   6        0.057s     0.723s .
  637923410906706577:   0        0.009s     0.732s 
  637923410906794680:   7        0.016s     0.748s 
  637923410906954930:   12       0.010s     0.758s 
  637923410907054826:   9        0.010s     0.768s 
  637923410907154554:   8        0.015s     0.783s 
  637923410907304984:   11       0.077s     0.860s .
  637923410908070135:   10       0.021s     0.881s 
  637923410908280116:   11       0.002s     0.883s 
  637923410908300126:   7        0.001s     0.884s 
  637923410908310123:   0        0.006s     0.890s 
  637923410908370118:   13       0.085s     0.974s .
  637923410909216586:   14       0.019s     0.993s 
  637923410909406574:   15       0.019s     1.012s 
  637923410909596489:   16       0.082s     1.094s .
  637923410910416570:   17       0.022s     1.116s 
  637923410910632660:   18       0.019s     1.135s 
  637923410910825705:   0        0.001s     1.136s 
  637923410910835445:   19       0.018s     1.155s 
  637923410911017863:   20       0.011s     1.166s 
  637923410911127953:   21       0.020s     1.186s 
  637923410911327571:   22       0.082s     1.268s .
  637923410912149437:   23       0.020s     1.288s 
  637923410912350002:   24       0.016s     1.304s 
  637923410912509413:   0        0.314s     1.618s ......
  637923410915840311:   25       0.005s     1.642s 
  637923410915890304:   26       0.006s     1.648s 
  637923410915950284:   28       0.281s     1.928s .....
  637923410918756923:   29       0.189s     2.118s ...
  637923410920647802:   30       0.089s     2.206s .
  637923410921536346:   32       0.094s     2.300s .
  637923410922472451:   30       0.002s     2.302s 
  637923410922492476:   31       0.071s     2.373s .
  637923410923203885:   28       0.001s     2.374s 
  637923410923213270:   26       0.001s     2.375s 
  637923410923223354:   27       0.002s     2.377s 
  637923410923243357:   25       0.001s     2.378s 
  -----------------------------------------------------------------------------------------------------------------------------------
  Utilization:          99.2     Average Utilization: 99.2
├ù Building Solution
- Build failed with message C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppBuild.targets(917,11): error MSB4057: The target "MarkupCompilePass1" does not exist in the project. [D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Common\Common.vcxproj]. Check your build configuration.
├ù Build failed with message C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppBuild.targets(917,11): error MSB4057: The target "MarkupCompilePass1" does not exist in the project. [D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Common\Common.vcxproj]. Check your build configuration.
- Gathering MSBuild data for telemetry.
i Gathering MSBuild data for telemetry.
- Loading properties from msbuildproperties.g.json
i Loading properties from msbuildproperties.g.json
Command failed with error MSBuildError: C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppBuild.targets(917,11): error MSB4057: The target "MarkupCompilePass1" does not exist in the project. [D:\Office Work\eescom_win\eescom\node_modules\react-native-windows\Common\Common.vcxproj]
Your telemetry sessionId was 2fefcbf6c6b008e2c4f542566dcc5eb8
`

also when I am trying to build it with a visual studio directly then I am getting same packages error 
![image](https://user-images.githubusercontent.com/41816068/176990801-54562c35-9095-46a9-a450-4fea861cf790.png)

**Comment by chrisglein** [comment, marked as answer]
Being further evaluated in the issue you created: #10234
